bundles/org.openhab.core.ephemeris/src/main/java/org/eclipse/smarthome/core/ephemeris/internal/EphemerisManagerImpl.java

```diff
@@ -75,17 +82,36 @@
     private static final String CONFIG_REGION = "region";
     private static final String CONFIG_CITY = "city";
 
+    static final String PROPERTY_COUNTRY_DESCRIPTION_PREFIX = "country.description.";
+    static final String PROPERTY_COUNTRY_DESCRIPTION_DELIMITER = "\\.";
+
+    static final List<ParameterOption> COUNTRIES = new ArrayList<>();
+    static final Map<String, List<ParameterOption>> REGIONS = new HashMap<>();
+    static final Map<String, List<ParameterOption>> CITIES = new HashMap<>();
```

Comment:
Do we really need to use static members on OSGi components that fill that information on activation?

I would prefer to use non static members.